examples/long_running_measurement/measurement.py

```diff
@@ -0,0 +1,103 @@
+"""Perform a long running measurement."""
+import logging
+import pathlib
+import random
+import threading
+import time
+from typing import Generator, List, Tuple
+
+import click
+import grpc
+
+import ni_measurementlink_service as nims
+
+service_directory = pathlib.Path(__file__).resolve().parent
+measurement_service = nims.MeasurementService(
+    service_config_path=service_directory / "LongRunningMeasurement.serviceconfig",
+    version="0.5.0.0",
+    ui_file_paths=[service_directory / "LongRunningMeasurement.measui"],
+)
+
+
+Outputs = Tuple[List[int], str, List[int], float]
+
+
+@measurement_service.register_measurement
+@measurement_service.configuration("time_in_seconds", nims.DataType.Int32, 10)
```

Comment:
Why is this an Int32? What would it mean to run this measurement for negative time?  :D   Same goes for random number and overlimit. As far as I can tell, those can only be positive integers and should probably be UInt32.

Reply:
I would prefer to use float to represent fractional seconds, but I copied the data types from @dixonjoel's C# version.

Random numbers are not inherently positive or integral, and I would be happy to replace them with a +/-10.0 sine wave or something.

Reply:
Yeah, that's fair. The random numbers that this particular example creates are always positive but that isn't guaranteed to always be true. But time in seconds should probably be a U32. Although if we change to a sine wave then we might want to be using periods instead, which I guess would also be positive.

Reply:
I'll post a separate PR with the revised example.